site/docs/pages/index.mdx

```diff
@@ -30,8 +30,9 @@ import TokenSelector from '../components/TokenSelector';
     <div className="md:text-center flex flex-col gap-8 pt-48 pb-16">
       <h1 className="text-center text-6xl md:text-9xl font-medium no-underline dark:text-white">OnchainKit</h1>
       <div className="text-center text-xl md:text-3xl dark:text-white">
-        <p className="leading-7 md:leading-none">React components
-        and TypeScript utilities to help you build top-tier onchain apps.</p>
+        <p className="leading-7 md:leading-none">
+          Build your onchain apps quickly with ready-to-use React components and TypeScript utilities.
+        </p>
```

Comment:
Updated text rather than what's in the mockup - `"Ship your onchain apps in minutes with out-of-the box React components and TypeScript utilities."`

Reply:
Let's just not change the text. I really like what we have. But thank you for trying.